### ISSUE TITLE
py_queries test is hanging in parallel

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#1222]
py_queries is hanging in parallel

I generated log files from a run, and I'm seeing divide by zero errors
on processor 1:

avtPythonQuery encountered EMPTY InputDataTree after ApplyFilters.  This
may be a valid state if running parallel and there are more processors
than domains.
avtPythonFilterEnvironment::Python Error - <type
'exceptions.ZeroDivisionError'>float division by zero
Traceback (most recent call last):
 File "<string>", line 26, in post_execute
ZeroDivisionError: float division by zero

Exception:
(VisItException) /g/g16/kbonnell/Branches/trunk/src/avt/Queries/Queries/avtPythonQuery.C, line 637: engine_par: avtPythonQuery::PostExecute Error - 'post_execute' call on python filter failed.
Python Environment Error:
<type 'exceptions.ZeroDivisionError'>float division by zero
Traceback (most recent call last):
 File "<string>", line 26, in post_execute
ZeroDivisionError: float division by zero

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Cyrus Harrison
Original creation: 07/01/2014 12:14 pm
Original update: 08/18/2014 07:07 pm
Ticket number: 1896
